numpy/core/_add_newdocs.py

```diff
@@ -2031,21 +2031,27 @@
     as well as documented private attributes):
 
     .. autoattribute:: numpy.core._internal._ctypes.data
+        :noindex:
 
     .. autoattribute:: numpy.core._internal._ctypes.shape
+        :noindex:
 
     .. autoattribute:: numpy.core._internal._ctypes.strides
+        :noindex:
 
     .. automethod:: numpy.core._internal._ctypes.data_as
+        :noindex:
 
     .. automethod:: numpy.core._internal._ctypes.shape_as
+        :noindex:
 
     .. automethod:: numpy.core._internal._ctypes.strides_as
+        :noindex:
```

Comment:
Adding `:noindex:` removed many warnings caused by manually documenting the attributes instead of autosummary documenting them. The pages render the same in html. Search still works, you get  [this](https://5997-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/search.html?q=_ctypes.data&check_keywords=yes&area=default) when searching for `_ctypes.data` vs. [this](http://www.numpy.org/devdocs/search.html?q=_ctypes.data) currently.